gallery/visualtests/nxCombobox.js

```diff
@@ -0,0 +1,210 @@
+    simpleTest,
+    blurElement,
+    disableLoadingSpinnerAnimation,
+    checkScreenshot,
+    a11yTest
+  } = setupBrowser('#/pages/Combobox');
+
+  const basicExampleSelector = '#nx-combobox-basic-example .nx-combobox',
+      nonEmptinessExampleSelector = '#nx-combobox-non-emptiness-example .nx-combobox',
+      backendExampleSelector = '#nx-combobox-backend-example .nx-combobox',
+      errorExampleSelector = '#nx-combobox-error-example .nx-combobox',
+      disabledExampleSelector = '#nx-combobox-disabled-example .nx-combobox';
+
+  describe('when loading', function() {
+    beforeEach(async function() {
+      const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+          loadingSpinnerSelector = `${backendExampleSelector} .nx-loading-spinner`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('1');
+      await getPage().waitForSelector(loadingSpinnerSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(backendExampleSelector);
+      await disableLoadingSpinnerAnimation();
+      await checkScreenshot(component, 300, 130);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  describe('when displaying results', function() {
+    beforeEach(async function() {
+      const inputSelector = `${backendExampleSelector} .nx-combobox__input input`,
+          dropdownButtonSelector = `${backendExampleSelector} .nx-dropdown-button`,
+          [input] = await waitAndGetElements(inputSelector);
+
+      await input.focus();
+      await getPage().keyboard.type('1');
+      await getPage().waitForSelector(dropdownButtonSelector);
+    });
+
+    it('looks right', async function() {
+      const [component] = await waitAndGetElements(backendExampleSelector);
+      await checkScreenshot(component, 300, 376);
+    });
+
+    it('passes a11Y checks', a11yTest());
+  });
+
+  it('looks right with long variant', async function() {
+    const inputSelector = `${basicExampleSelector} .nx-combobox__input input`,
+        dropdownButtonSelector = `${basicExampleSelector} .nx-dropdown-button`,
+        [component, input] = await waitAndGetElements(basicExampleSelector, inputSelector);
+
+    await input.focus();
+    await getPage().keyboard.type('a');
+    await getPage().waitForSelector(dropdownButtonSelector);
+    await checkScreenshot(component, 800, 376);
+  });
+
+  it('looks right with short variant', async function() {
+    const inputSelector = `${nonEmptinessExampleSelector} .nx-combobox__input input`,
+        dropdownButtonSelector = `${nonEmptinessExampleSelector} .nx-dropdown-button`,
+        [component, input] = await waitAndGetElements(nonEmptinessExampleSelector, inputSelector);
+
+    await input.focus();
+    await getPage().keyboard.type('a');
+    await getPage().waitForSelector(dropdownButtonSelector);
+    await checkScreenshot(component, 150, 376);
```

Comment:
This is ending up identical to the success validation screenshot, so we don't need both.

Reply:
Removed here 19c5b7eebbe4f4c81fdfe80b34dc05ed35b1cba2.